### PR TITLE
New version: prefix-dev.pixi version 0.34.0

### PR DESCRIPTION
**Pull request by ruben-arts**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/184932)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 897.)

**Comment by traversaro** [comment]
> Hello @ruben-arts,
> 
> This pull request has been automatically marked as stale because it has been marked as requiring author feedback but has not had any recent activity. It will be closed if no further activity occurs **within 3 days of this comment**.
> 
> Template: msftbot/noRecentActivity

fyi @ruben-arts, I do not have experience with winget, but do you think something could be done for this? Thanks!

**Comment by ruben-arts** [comment]
Apparently you triggered a rerun @traversaro, so hopefully it will go in soon. I don't see why it would not succeed and those errors make no sense. 

Thanks for the ping though, these winget PR's almost always fail and then resolve themself so I got blind for their emails.

File: manifests/p/prefix-dev/pixi/0.34.0/prefix-dev.pixi.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+UpgradeBehavior: install
+ProductCode: '{47A0AAE8-0719-4F57-A9ED-B0DC956CB89E}'
+ReleaseDate: 2024-10-21
+AppsAndFeaturesEntries:
+- DisplayVersion: 0.34.0
+  UpgradeCode: '{D983A0B8-0183-4F99-BA13-A1C49BD8E46F}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/prefix-dev/pixi/releases/download/v0.34.0/pixi-x86_64-pc-windows-msvc.msi
+  InstallerSha256: 3411D707B9E93D5F2726FEC5F6D6D1F7A9715D049FEBC67D0D909BAC2E6FCC03
```
Comment (stephengillie) [inline manifests/p/prefix-dev/pixi/0.34.0/prefix-dev.pixi.installer.yaml:19]:
```suggestion
  InstallerSha256: 60170A28BDB2560F62819EE3165A8A3A7F2ABD3BAEFBB7DEA5E2CD6DD608E49C
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Automatic Validation ended with:
> Actual hash:   60170A28BDB2560F62819EE3165A8A3A7F2ABD3BAEFBB7DEA5E2CD6DD608E49C
Expected hash: 3411d707b9e93d5f2726fec5f6d6d1f7a9715d049febc67d0d909bac2e6fcc03

**Pull request closed without merging**